src/index.js

```diff
@@ -0,0 +1,203 @@
+        state.backgroundImage = images.backgroundSnowing;
+    }
+    skyAccessible.textContent = state.skyAccessible;
+    skyBackground.style.backgroundImage = `url('${state.backgroundImage}')`;
+};
+
+const changeCity = () => {
+    const city = document.getElementById('city');
+    const cityName = document.getElementById('cityName');
+    cityName.textContent = city.value;
+};
+
+const reset = () => {
+    const city = document.getElementById('city');
+    const cityName = document.getElementById('cityName');
+    const temp = document.getElementById('temp');
+    const foregroundAccessible = document.getElementById(
+        'weatherGardenAccessibleForeground'
+    );
+    const skyAccessible = document.getElementById('weatherGardenAccessibleSky');
+    const skyBackground = document.getElementById('sky');
+    const foreground = document.getElementById('weatherGarden');
+    state.city = 'Baltimore';
+    state.temp = 70;
+    state.tempColour = 'black';
+    state.foregroundAccessible = accessibleText.foregroundSpring;
+    state.skyAccessible = accessibleText.backgroundSunny;
+    state.backgroundImage = images.backgroundSunny;
+    state.foreground = images.foregroundSpring;
+    city.value = state.city;
+    cityName.textContent = state.city;
+    temp.textContent = `${state.temp}°F`;
+    temp.style.color = state.tempColour;
+    foregroundAccessible.textContent = state.foregroundAccessible;
+    skyAccessible.textContent = state.skyAccessible;
+    skyBackground.style.backgroundImage = `url('${state.backgroundImage}')`;
+    foreground.src = state.foreground;
+};
+
+const registerEventHandlers = () => {
+    const up = document.getElementById('up');
+    up.addEventListener('click', increaseTemp);
+    const down = document.getElementById('down');
+    down.addEventListener('click', decreaseTemp);
+    const sky = document.getElementById('sky-select');
+    sky.addEventListener('change', changeSky);
+    const city = document.getElementById('city');
+    city.addEventListener('change', changeCity);
+    const getWeatherButton = document.getElementById('getWeatherButton');
+    getWeatherButton.addEventListener('click', getLocation);
+    const resetButton = document.getElementById('resetButton');
+    resetButton.addEventListener('click', reset);
+};
+
+document.addEventListener('DOMContentLoaded', registerEventHandlers);
+
+const convertKelvinToFahrenheit = (kelvin) => {
+    return Math.round((kelvin - 273.15) * 1.8 + 32);
+};
+
+const getLocation = () => {
+    const city = document.getElementById('cityName');
+    const cityName = city.textContent;
+    axios
+        .get('http://localhost:5000/location', {
+            params: {
+                q: cityName,
+            },
+        })
+        .then((response) => {
+            const latitude = response.data[0].lat;
+            const longitude = response.data[0].lon;
+            getWeather(latitude, longitude);
```

Comment:
Nice splitting the weather call out to its own function to keep this function shorter & easier to read.